### PR TITLE
Update display name for Azure Load Testing

### PR DESCRIPTION
**Pull request by ninallam**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -5,7 +5,7 @@
 "Azure.Extensions.AspNetCore.DataProtection.Blobs","1.2.3","","ASP.NET Extension - DataProtection Blobs","Extensions","extensions","","","client","true","","09/12/2022","06/05/2020","active","","","","","","",""
 "Azure.Extensions.AspNetCore.DataProtection.Keys","1.1.0","","ASP.NET Extension - DataProtection Keys","Extensions","extensions","","","client","true","","09/02/2021","06/05/2020","active","","","","","","",""
 "Azure.Security.Attestation","1.0.0","","Attestation","Attestation","attestation","","","client","true","","05/07/2021","05/07/2021","","","","","","","",""
-"Azure.Developer.LoadTesting","","1.0.0-beta.1","Azure Loadtesting Service ","Loadtestservice","loadtestservice","NA","","client","true","","","","","","","","","","",""
+"Azure.Developer.LoadTesting","","1.0.0-beta.1","Azure Load Testing","Loadtestservice","loadtestservice","NA","","client","true","","","","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:8]:
```suggestion
"Azure.Developer.LoadTesting","","1.0.0-beta.1","Load Testing","Load Test Service","loadtestservice","NA","","client","true","","","","","","","","","","",""
```

We generally exclude Azure in the service name as it is usually implied.

**Comment by weshaggard** [comment]
We should also be sure to update the other language csv entries with a consistent Service name and Display name.

**Comment by ninallam** [comment]
@weshaggard I've updated the name to be consistent with all other languages.

**Comment by weshaggard** [comment]
I think there is still some inconsistencies in Java https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/java-packages.csv#L6

**Review by weshaggard — APPROVED** [review]
Please also update the Java package list to match but otherwise looks good.

**Comment by ninallam** [comment]
@weshaggard thanks I've created a PR for Java as well.

**Comment by ninallam** [comment]
#sign-off

**Comment by ninallam** [comment]
@czubair can you please help merge the changes?

**Review by czubair — APPROVED** [review]

**Pull request merged**